charts/apps/kubeflow-pipelines/rds-only/templates/Deployment/ml-pipeline-viewer-crd-kubeflow-Deployment.yaml

```diff
@@ -29,14 +29,13 @@ spec:
       - env:
         - name: NAMESPACE
           value: ''
-          valueFrom: null
```

Comment:
can you confirm this was a bug previously?

Reply:
kustomize version issue. doesn't affect behavior, it;s null